src/staverepetition.ts

```diff
@@ -101,95 +99,52 @@ export class Repetition extends StaveModifier {
   }
 
   drawCodaFixed(stave: Stave, x: number): this {
-    const y = stave.getYForTopText(stave.getNumLines()) + this.yShift;
-    Glyph.renderGlyph(
-      stave.checkContext(),
-      this.x + x + this.xShift,
-      y + Tables.currentMusicFont().lookupMetric('staveRepetition.coda.offsetY'),
-      40,
-      'coda',
-      { category: 'coda' }
-    );
+    const y = stave.getYForTopText(stave.getNumLines());
+    this.text = '\ue048' /*coda*/;
+    this.renderText(stave.checkContext(), x, y + Tables.lookupMetric('Repetition.coda.offsetY'));
     return this;
   }
 
   drawSignoFixed(stave: Stave, x: number): this {
```

Comment:
Since we're here, can we rename this method? It misspells it as signo.

drawSignoFixed => drawSegnoFixed

Reply:
This typo was in version 3.0.9, so we should document this breaking change (along with all our other breaking changes). :-)

https://github.com/0xfe/vexflow/blob/3.0.9/src/staverepetition.js#L95

Reply:
OK, I created a wiki page to document breaking changes :-)

https://github.com/vexflow/vexflow/wiki/Breaking-Changes

Reply:
Done. 
Is this really a breaking change? The method is only used internally. We have not started protecting internal member functions. Should we?

Reply:
For more flexibility, I prefer not protecting internal member functions unless there is a good reason for it to be private.

The reason is that a developer can subclass / override methods if we don't protect them. Or they can do something cool / unexpected by calling our internal methods. That makes the library more useful in general. 

But if we make every method private, then developers are limited in what they can do (unless they download from the main branch and edit the code themselves).

Reply:
Also, even if it is not a "breaking change" by strict definition, I figure it doesn't hurt to document this somewhere, in case someone else asks later on.

Reply:
@ronyeh I meant to make methods protected, not private. Protected methods can be overriden.